recipes/server_setup.rb

```diff
@@ -114,8 +114,9 @@
       case volume_values['volume_type']
       when 'replicated'
         # Ensure the trusted pool has the correct number of bricks available
-        if brick_count < volume_values['replica_count']
-          Chef::Log.warn("Correct number of bricks not available for volume #{volume_name}. Skipping...")
+        requiredBricks = (volume_values['replica_count'] * volume_values['peers'].count)
+        if brick_count != requiredBricks
```

Comment:
Can you elaborate on this?

If i am reading it right this would not work.  With an RF of 3 with 3 nodes so 1 brick per node, `requiredBricks` would be 9 but you only have 3 bricks.